google-home-community.groovy

```diff
@@ -2122,6 +2090,15 @@ private controlScene(options) {
     device."${options.command}"()
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
```

Comment:
This method is used directly, you shouldn't need to suppress an unused warning.

Reply:
Done!